.travis.yml

```diff
@@ -85,6 +81,16 @@ script: >
     banner "git check" &&
     git diff --check $(git rev-list HEAD | tail -1) &&
 
+    # ensure only two uses of jerry_string_to_char_buffer in zjs_util.c
```

Comment:
In the future we'll want to update this so we can show the trlite error message.  Otherwise it won't be clear why the build failed to users unfamiliar with our rule set.